sklearn/decomposition/kernel_pca.py

```diff
@@ -81,24 +81,36 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         A pseudo random number generator used for the initialization of the
         residuals when eigen_solver == 'arpack'.
 
-    n_jobs : int, optional (default = 1)
+    n_jobs : int, default=1
         The number of parallel jobs to run.
-        If ``-1``, then the number of jobs is set to the number of CPU cores.
+        If `-1`, then the number of jobs is set to the number of CPU cores.
+
+    copy_X : boolean, default=True
+        If True, input X is copied and stored by the model. If no further
+        changes will be done to X, setting `copy_X=False` saves memory by
+        storing a reference.
 
     Attributes
     ----------
+    lambdas_ : array, (n_components,)
+        Eigenvalues of the centered kernel matrix in decreasing order.
+        If `n_components` and `remove_zero_eig` are not set,
+        then all values are stored.
+
+    alphas_ : array, (n_samples, n_components)
+        Eigenvectors of the centered kernel matrix. If `n_components` and
+        `remove_zero_eig` are not set, then all components are stored.
 
-    lambdas_ :
-        Eigenvalues of the centered kernel matrix
+    dual_coef_ : array, (n_samples, n_features)
+        Inverse transform matrix. If `fit_inverse_transform=False`,
+        dual_coef_=None
 
-    alphas_ :
-        Eigenvectors of the centered kernel matrix
+    X_transformed_fit_ : array, (n_samples, n_components)
+        Projection of the fitted data on the kernel principal components.
 
-    dual_coef_ :
-        Inverse transform matrix
+    X_fit_ : (n_samples, n_features)
+        A copy of the original fitted data. If `copy_X=False`, X_fit_=None.
```

Comment:
It looks like ``X_fit_`` will no longer be ``None``. I'd just say ``the data used to fit the model``.